grouper/fe/handlers/group_request_update.py

```diff
@@ -92,6 +92,31 @@ def post(self, request_id, group_id=None, name=None):
         edge = self.session.query(GroupEdge).filter_by(
             id=request.edge_id
         ).one()
+
+        approver_mail_to = [
+            user.name
+            for user in group.my_users()
+            if GROUP_EDGE_ROLES[user.role] in ("manager", "owner", "np-owner") and
```

Comment:
you should use the `APPROVER_ROLE_INDICIES` set in `model_soup` instead of defining the set of roles inline. you'd need to use `user._role` instead.